tests/TestHelpers/GraphHelper.php

```diff
@@ -1426,4 +1426,45 @@ public static function unassignRole(
 			self::getRequestHeaders(),
 		);
 	}
+
+	/**
+	 * @param string $baseUrl
+	 * @param string $xRequestId
+	 * @param string $user
+	 * @param string $password
+	 * @param string $path
+	 *
+	 * @return string
+	 * @throws GuzzleException
+	 * @throws Exception
+	 */
+	public static function getMountPointId(
```

Comment:
In issue https://github.com/owncloud/ocis/issues/6737, we use `fileId` in the endpoint, and share id or mount point id is not necessary.
In this PR we can use either share id or mount point id to preview files. Viktor recommended using mount point id instead of share id.